build.rs

```diff
@@ -10,6 +10,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     println!("cargo:rerun-if-changed=res/proto/google/cloud/texttospeech/v1/cloud_tts.proto");
     println!("cargo:rerun-if-changed=res/proto/google/cloud/texttospeech/v1beta1/cloud_tts.proto");
 
+    let skip_bindgen = std::env::var("SKIP_PROTO_FILES_COMPILATION")
```

Comment:
just to make it perfect can we also rename the variable **skip_proto_compilation** instead of s**kip_bindgen**. ?

thanks!

Reply:
Ha, youre right! Just had another go at it